src/trans.rs

```diff
@@ -534,6 +532,16 @@ impl<'v, 'tcx: 'v> BinaryenFnCtxt<'v, 'tcx> {
                 TerminatorKind::Return => {
                     /* handled during bb creation */
                 }
+                TerminatorKind::Assert { ref target, .. } => {
+                    // TODO: An assert is not a GOTO!!!
```

Comment:
It's a [conditional call to one of two lang items](https://github.com/rust-lang/rust/blob/1284081d33d0e7856c6d242b3e311876c4eec048/src/librustc_trans/mir/block.rs#L312).